docs/guide/learning/how-to-learn-vue.md

```diff
@@ -243,6 +243,256 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### Laravel
 
+#### Reasons to use Laravel and Vue
+
+##### Scaffolded by default
+
+Laravel [by default comes with everything you need](https://laravel.com/docs/5.7/frontend#writing-vue-components) to get up and running with Vue.
+
+After having created a Laravel project, you'll be provided with:
```

Comment:
This is too detailed. You could just say it comes out of the box with Vue setup to work as an enhancement over normal server rendered apps, i.e. with Global Components.

Reply:
Okay, I was worried I was probably going into too much detail!

I'll see what I can do to shorten it.